src/coreclr/jit/optimizer.cpp

```diff
@@ -2580,6 +2580,33 @@ void Compiler::optFindNaturalLoops()
 #endif // DEBUG
 }
 
+//-----------------------------------------------------------------------------
+//
+// All the inner loops that whose block weight meets a threshold are marked
+// as needing alignment.
+//
+
+void Compiler::optIdentifyLoopsForAlignment()
+{
+#ifdef FEATURE_LOOP_ALIGN
+    if (codeGen->ShouldAlignLoops())
+    {
+        for (unsigned char loopInd = 0; loopInd < optLoopCount; loopInd++)
+        {
+            BasicBlock* first = optLoopTable[loopInd].lpFirst;
+
+            // An innerloop candidate that might need alignment
+            if ((optLoopTable[loopInd].lpChild == BasicBlock::NOT_IN_LOOP) &&
+                opts.compJitAlignLoopMinBlockWeight <= first->getBBWeight(this))
```

Comment:
Seems clearer this way?
```suggestion
                first->getBBWeight(this) >= opts.compJitAlignLoopMinBlockWeight)
```